src/server/pps/server/api_server.go

```diff
@@ -1730,6 +1741,19 @@ func jobStateToStopped(state pps.JobState) bool {
 	}
 }
 
+func (a *apiServer) getPachClient() (*client.APIClient, error) {
```

Comment:
If we have this, we should remove code that uses `a.pachConn`.  Code that uses API clients directly (such as `pfs.APIClient`) could use `a.pachClient.PfsAPIClient` instead.  We shouldn't have both `a.pachClient` and `a.pachConn` (and their `Once`s).